tools/barista/src/builder/icons.ts

```diff
@@ -70,7 +64,9 @@ function getIconFilesByPath(rootPath: string): string[] {
  * Get the changelog template containing all changes for each icon sorted by iconpack version.
  */
 function getChangelogTemplate(): BaIconsChangelog {
-  const changelogContent = readFileSync(CHANGELOG_SRC).toString();
+  const changelogContent = readFileSync(
```

Comment:
if you pass write `readFileSync(environment.iconsChangelogFileName, { encoding: 'utf-8' })` you don't need a to string ;) but check if the file exists first before you try to read it.